src/entrypoints/my-redirect.ts

```diff
@@ -19,6 +19,12 @@ const createRedirectParams = (): string => {
   const redirectParams = window.redirect.params;
   const userParams = extractSearchParamsObject();
   delete userParams.mobile;
+  delete userParams.dclid;
```

Comment:
😆 

I wonder if we should maybe not compare length as a validation.

Reply:
Then we should no longer check if all params that are passed in are valid, but only check the params that are defined in the redirect.